summarization/summarizer_main.py

```diff
@@ -65,6 +63,9 @@ def get_summarized_content(self):
         ''' Summarizes the contents of the
         webpage and returns it as a StampPage object
         '''
+        # strip title text
+        self._strip_title_and_make_stamp()
+
```

Comment:
The title contents list was empty since we were testing the image caption pruning method.
This method when called on an empty list tried to access an element which caused an error